### ISSUE TITLE
"Sorry, an error occurred while processing your request."

### ISSUE THREAD

**Issue by tomhunter-gh** [sethwebster/TweetHarbor#18]
I was looking for some kind of notification system for AppHarbor builds and TweetHarbor looks like a great tool.  I would love to get this set up for SourceLog (https://github.com/tomhunter-gh/SourceLog).

Unfortunately I'm getting the above error after I authorize TweetHarbor to use my Twitter account. Twitter is re-directing back to this page which reports the error (I've removed query string parameter values):
http://tweetharbor.apphb.com/Account/OAuthComplete/?Client=twitter&oauth_token=XXX&oauth_verifier=XXX

I get the same issue if I go via http://tweetharbor.com/

**Comment by sethwebster** [comment]
Yeah, sorry @tomhunter-gh ... I've been swamped @ day job and cross country move... I'll try to take a look at this soon. Accept my apologies for the delay.

**Comment by codemonkey85** [comment]
Just an FYI - this is happening to me as well right now.

**Comment by sethwebster** [comment]
Sorry, @codemonkey85  -- been crazy trying to get time for side projects for a while now. I'll see what I can do this week.
